src/loggedfs.cpp

```diff
@@ -667,12 +667,14 @@ static int loggedFS_release(const char *path, struct fuse_file_info *fi)
 static int loggedFS_fsync(const char *path, int isdatasync,
                           struct fuse_file_info *fi)
 {
-    /* Just a stub.  This method is optional and can safely be left
-       unimplemented */
-
-    (void) path;
-    (void) isdatasync;
+    char *aPath=getAbsolutePath(path);
```

Comment:
if you want to log release, this code should be in loggedFS_release and loggedFS_fsync no ?

Reply:
I added the log